.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    types: [ opened, synchronize ]
```

Comment:
I've never seen this used before. The docs state:

> **Note:** By default, a workflow only runs when a `pull_request`'s activity type is `opened`, `synchronize`, or `reopened`. To trigger workflows for more activity types, use the `types` keyword.

So I would think the default should be sufficient, probably with a branch filter to match the `push:` filter.

Reply:
> probably with a branch filter to match the `push:` filter.

I would have thought we would want it for all PRs, regardless of branch.